frontend/src/component/project/Project/ProjectLog/ProjectLog.styles.ts

```diff
@@ -0,0 +1,9 @@
+import { makeStyles } from 'tss-react/mui';
+
+export const useStyles = makeStyles()(theme => ({
+    container: {
+        borderRadius: '12.5px',
+        backgroundColor: theme.palette.background.paper,
+        padding: '2rem',
+    },
+}));
```

Comment:
@aneeshrelan We're migrating away from this kind of styling, since the migration to MUI 5 this has been deprecated and is kept alive through a module interop. We should either use styled components, or the built in sx propperty in material ui to style. Using useStyles adds technical debt that we need to clean up later.